### PR TITLE
Miscellaneous cleanup

### PR DESCRIPTION
**Pull request by andyscott**
Various cleanup. Feel free to merge if you want any of this now.

### CODE REVIEW

**Comment by dialelo** [comment]
Looks good to me :+1:

File: common/Dependencies.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import sbt.Keys._
+import sbt._
+
+trait Dependencies {
+
+  lazy val scalatest = "org.scalatest" %% "scalatest" % "3.0.0-M15"
+
+  object cats {
+    lazy val Version  = "0.4.1"
```
Comment (raulraja) [inline common/Dependencies.scala]: Any reason why these need to be lazy?

**Comment by raulraja** [comment]
LGTM!

File: common/Dependencies.scala (hunk 1 above)
Comment (andyscott) [inline common/Dependencies.scala]: Nope. Just kind of habit.

**Pull request merged**